src/ECS/MessageTypes.h

```diff
@@ -9,6 +9,8 @@
         #define PACK( __Declaration__ ) __pragma( pack(push, 1) ) __Declaration__ __pragma( pack(pop))
     #endif
 
+typedef unsigned int n_id_t;
```

Comment:
exist in Nitwork.h

Reply:
nop, j'lai suprr

Reply:
par ce que nitwork.h inclus MessageTypes.h